test/test-internal-limit.js

```diff
@@ -0,0 +1,68 @@
+/* global describe, it, */
+var assert = require("assert");
+var cc = require("../lib/countly-common");
+
+// Unit tests for truncateSingleValue:
+describe("Internal limits single value truncation testing suite", function() {
+    it("Check if the string is truncated", function() {
+        var string = "123456789";
+        var limit = 3;
+        var errorLog = "test";
+        var newStr = cc.truncateSingleValue(string, limit, errorLog);
+        assert.equal(newStr.length, 3);
+    });
+    it("Check if the number is truncated", function() {
+        var number = 123456789;
+        var limit = 3;
+        var errorLog = "test";
+        var newStr = cc.truncateSingleValue(number, limit, errorLog);
+        assert.equal(newStr.length, 3);
```

Comment:
verify the resulting string and not just it's length. Theoretically the length might match but the content might be garbage